MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -120,6 +137,13 @@ public void onItemClicked(RecyclerView recyclerView, int position, View view) {
           return;
         }
         startActivity(exampleItemModel.get(position).getActivity());
+
+        AnalyticsTracker.getInstance(getApplicationContext()).clickedOnIndividualExample(
```

Comment:
Considering the number of times that you're calling to `AnalyticsTracker.getInstance()` why don't you store the instance in the activity?